### ISSUE TITLE
Build out structure

### ISSUE THREAD

**Issue by smit1678** [healthsites/sources#1]
Thinking the structure for the repo will be like this: 

```
|--country/
    |--data.csv
    |--license.txt
    |--readme.txt
    |--scripts/
```

Idea would be that we'd start tracking data that can be imported, license connected to it, readme with other details, and any custom scripts that were used to clean or manipulate the data into the import structure. 

Other input here @markherringer?

**Comment by markherringer** [comment]
Do we want to collect the data if it's not already available under an ODbL ?

**Comment by smit1678** [comment]
Yeah, I imagine there will be a lot of existing open data out there that is not tagged ODBL. We'll want to track and understand if the license is compatible with ODBL or not. This repo will be a prep repo for data. This will enable us to track changes as data is organized and ready for import. Essentially this will move us away from using the wiki pages and into this repo to track the progress of an import.
